.dotnet/tests/Embeddings/OpenAIEmbeddingsModelFactoryTests.cs

```diff
@@ -3,114 +3,113 @@
 using NUnit.Framework;
 using OpenAI.Embeddings;
 
-namespace OpenAI.Tests.Embeddings
+namespace OpenAI.Tests.Embeddings;
```

Comment:
I'm just nesting out everything that was inside the `namespace` block to keep the same pattern other files have. No changes were made to the tests.